javascript/javascript1/week3/seriesDurations.js

```diff
@@ -0,0 +1,38 @@
+const seriesDurations = [
+    {
+      title: "Lost",
+      days: 3,
+      hours: 16,
+      minutes: 30,
+    },
+    {
+      title: "Money Heist",
+      days: 5,
+      hours: 17,
+      minutes: 1,
+    },
+    {
+      title: "Prison Break",
+      days: 8,
+      hours: 11,
+      minutes: 1,
+    },
+  ];
+  const averageLifeSpanMinutes = 80 * 365 * 24 * 60;
```

Comment:
Remember you can convert these numbers to variables to avoid [Magic numbers antipattern](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants)
Ex: `DAYS_IN_A_YEAR = 365`

Reply:
On the other hand, good job naming the variable :+1: